CouchbaseLite.xcodeproj/project.pbxproj

```diff
@@ -5246,6 +5247,7 @@
 				9343F192207D636300F19A89 /* DatabaseTest.swift in Sources */,
 				9343F193207D636300F19A89 /* CBLTestHelper.m in Sources */,
 				1AA91DC722B0356000BF0BDE /* CustomLogger.swift in Sources */,
+				69845B0723354D0A00CC16BB /* DateTimeQueryFunctionTest.swift in Sources */,
```

Comment:
Is this a new test?

Reply:
No, it was an old one, but not added to EE test target.